rules.ini

```diff
@@ -32,6 +32,8 @@ WAD = \.wad$ ; idTech1
 Adobe_AIR = (?:^|/)Adobe AIR(?:$|/)
 AdventureGameStudio = (?:^|/)(?:AGSteam\.dll|acsetup\.cfg)$
 AGI = (?:^|/)AGIDATA\.OVL$
+BlenderGameEngine[] = (?:^|/)BlendThumb(?:64|)\.dll$
```

Comment:
This might be too generic and is probably what makes it match Blender or [Motion Studio](https://steamdb.info/app/1534490/) which are not Blender Game Engine nor UPBGE applications.